src/client/Microsoft.Identity.Client/Platforms/netdesktop/Http/NetDesktopHttpClientFactory.cs

```diff
@@ -10,6 +10,9 @@ namespace Microsoft.Identity.Client.Platforms.net45.Http
 {
     internal class NetDesktopHttpClientFactory : IMsalHttpClientFactory
     {
+        //Prior to the changes needed in order to make MSAL's httpClients thread safe (https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/2046/files),
+        //the httpClient had the possibility to throw an exception stating "Properties can only be modified before sending the first request".
+        //MSAL's httpClient will no longer throw this exception after 4.19.0 (https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/releases/tag/4.19.0)
```

Comment:
what about an aka.ms link w/the info?

Reply:
and how will a customer find this information? I suggest the information is available where we talk about bringing your own httpclient. and should probably be information available in the reference doc where a customer can specify the httpclientfactory as a remark (with the aka.ms link)

---
In reply to: [526570158](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/2211#discussion_r526570158) [](ancestors = 526570158)